### PR TITLE
Fix for #997 - set an option to install latest SDK

### PR DESCRIPTION
**Pull request by shapirov103**
*Issue #, if available:* #997

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by elamaran11 — APPROVED** [review]
LGTM

**Comment by shapirov103** [comment]
Failed with `Received response status [FAILED] from custom resource. Message returned: Cannot delete because cluster blueprint-construct-dev currently has update ed48a6c0-4b5e-492d-ae4e-f953bfecf921 in progress `

File: lib/utils/cluster-utils.ts [hunk 1]
```diff
@@ -1,82 +1,7 @@
-import * as eks from "aws-cdk-lib/aws-eks";
-import { Stack } from "aws-cdk-lib";
-import { AwsCustomResource, AwsCustomResourcePolicy } from "aws-cdk-lib/custom-resources";
 import { Construct } from "constructs";
 import * as customResource from 'aws-cdk-lib/custom-resources';
 import { ClusterInfo } from "../spi";
 
-// Available Control Plane logging types
-const CONTROL_PLANE_LOG_TYPES = ['api','audit','authenticator','controllerManager','scheduler'];
```
Comment (elamaran11) [inline lib/utils/cluster-utils.ts:9]: Why are we removing this?

**Review by elamaran11 — COMMENTED** [review]
One minor comment @shapirov103

File: lib/utils/cluster-utils.ts (hunk 1 above)
Comment (elamaran11) [reply lib/utils/cluster-utils.ts:9]: Make sense.

**Pull request merged**